addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -824,11 +824,15 @@ registry.backgroundVideo = publicWidget.Widget.extend({
         // YouTube does not allow to auto-play video in mobile devices, so we
         // have to play the video manually.
         if (this.isMobileEnv && this.isYoutubeVideo) {
-            new window.YT.Player(this.iframeID, {
-                events: {
-                    onReady: ev => ev.target.playVideo(),
-                }
+            var iframeID = this.iframeID;
```

Comment:
You can skip this if you use arrow function, `this` will remain unchanged, so you can access `this.iframeID` 2 lines below